File: Android Java Retrofit Node Express- Orphanslife/app/src/main/java/com/kaligotla/oms/AdminView/Role/Role.java
package com.kaligotla.oms.AdminView.Role;

import java.io.Serializable;

public class Role implements Serializable {

    private int id;

    private String role, created_at, updated_at;

    public Role() {
    }

    public Role(int id) {
        this.id = id;
    }

    public Role(String role) {
        this.role = role;
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getRole() {
        return role;
    }

    public void setRole(String role) {
        this.role = role;
    }

    public String getCreated_at() {
        return created_at;
    }

    public void setCreated_at(String created_at) {
        this.created_at = created_at;
    }

    public String getUpdated_at() {
        return updated_at;
    }

    public void setUpdated_at(String updated_at) {
        this.updated_at = updated_at;
    }

    @Override
    public String toString() {
        return "Role{" +
                "id=" + id +
                ", role='" + role + '\'' +
                ", created_at='" + created_at + '\'' +
                ", updated_at='" + updated_at + '\'' +
                '}';
    }
}

File: Android Kotlin Retrofit Node Express- Orphanslife/app/src/main/java/com/kaligotla/orphanslife/data/model/Role.kt
package com.kaligotla.orphanslife.data.model

import java.io.Serializable

class Role : Serializable {
    var id = 0
    var role: String? = null
    var created_at: String? = null
    var updated_at: String? = null

    data class Role(
        var id: Int,
        var role: String,
        var created_at: String,
        var updated_at: String,
    )

    constructor() {}
    constructor(id: Int) {
        this.id = id
    }

    constructor(role: String?) {
        this.role = role
    }
}
